aiohttp_session/redis_storage.py

```diff
@@ -43,9 +43,13 @@ def save_session(self, request, response, session):
             self.save_cookie(response, key,
                              max_age=session.max_age)
         else:
-            key = str(key)
-            self.save_cookie(response, key,
-                             max_age=session.max_age)
+            if session.empty:
+                self.save_cookie(response, session._mapping,
```

Comment:
Ah yes now I remembered why I used session._mapping specifically :P It's what is already done in `cookie_storage.py` to delete the cookie (see [here](https://github.com/aio-libs/aiohttp-session/blob/master/aiohttp_session/cookie_storage.py#L48-L50))